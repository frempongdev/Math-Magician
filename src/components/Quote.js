import QuoteStyle from '../styleModules/Quote.module.css';

const Quote = () => (
  <div className={QuoteStyle.quoteBox}>
    <p>
      Mathematics is not about numbers, equations, computations, or algorithms:
      it is about understanding. â€” William Paul Thurston
    </p>
  </div>
);

export default Quote;
