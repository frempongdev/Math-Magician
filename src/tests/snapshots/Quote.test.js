import renderer from 'react-test-renderer';
import Quote from '../../components/Quote.js';// eslint-disable-line import/extensions

describe('Quote', () => {
  test('should match with Quote snapshot', () => {
    const tree = renderer.create(
      <Quote />,
    );
    expect(tree).toMatchSnapshot();
  });
});
