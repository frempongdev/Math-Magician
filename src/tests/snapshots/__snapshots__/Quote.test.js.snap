// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote should match with Quote snapshot 1`] = `
<div
  className="quoteBox"
>
  <p>
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. â€” William Paul Thurston
  </p>
</div>
`;
